content/id/docs/contribute/localization_id.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Dokumentasi Khusus Untuk Translasi Bahasa Indonesia
+content_type: concept
+---
+
+<!-- overview -->
+
+Panduan khusus untuk bergabung ke komunitas SIG DOC Indonesia dan melakukan 
+kontribusi untuk mentranslasikan dokumentasi Kubernetes ke dalam Bahasa 
+Indonesia.
+
+<!-- body -->
+
+## Manajemen _Milestone_ Tim {#manajemen-milestone-tim}
+
+Secara umum siklus translasi dokumentasi ke Bahasa Indonesia akan dilakukan 
+3 kali dalam setahun (sekitar setiap 4 bulan). Untuk menentukan dan mengevaluasi 
+pencapaian atau _milestone_ dalam kurun waktu tersebut [jadwal rapat daring 
+reguler tim Bahasa Indonesia](https://zoom.us/j/6072809193) dilakukan secara 
+konsisten setiap dua minggu sekali. Dalam [agenda rapat ini](https://docs.google.com/document/d/1Qrj-WUAMA11V6KmcfxJsXcPeWwMbFsyBGV4RGbrSRXY) 
+juga dilakukan pemilihan PR _Wrangler_ untuk dua minggu ke depan. Tugas PR 
+_Wrangler_ tim Bahasa Indonesia serupa dengan PR _Wrangler_ dari proyek 
+_upstream_.
+
+Target pencapaian atau _milestone_ tim akan dirilis sebagai 
+[_issue tracking_ seperti ini](https://github.com/kubernetes/website/issues/22296) 
+pada Kubernetes GitHub Website setiap 4 bulan. Dan bersama dengan informasi 
+PR _Wrangler_ yang dipilih setiap dua minggu, keduanya akan diumumkan di Slack 
+_channel_ [#kubernetes-docs-id](https://kubernetes.slack.com/archives/CJ1LUCUHM) 
+dari Komunitas Kubernetes.
+
+## Cara Memulai Translasi
+
+Untuk menerjemahkan satu halaman Bahasa Inggris ke Bahasa Indonesia, lakukan 
+langkah-langkah berikut ini:
+
+* Check halaman _issue_ di GitHub dan pastikan tidak ada orang lain yang sudah 
+mengklaim halaman kamu dalam daftar periksa atau komentar-komentar sebelumnya.
+* Klaim halaman kamu pada _issue_ di GitHub dengan memberikan komentar di bawah 
+dengan nama halaman yang ingin kamu terjemahkan dan ambillah hanya satu halaman 
+dalam satu waktu.
+* _Fork_ [repo ini](https://github.com/kubernetes/website), buat terjemahan 
+kamu, dan kirimkan PR (_pull request_) dengan label `language/id`
+* Setelah dikirim, pengulas akan memberikan komentar dalam beberapa hari, dan 
+tolong untuk menjawab semua komentar. Direkomendasikan juga untuk melakukan 
+[_squash_](https://github.com/wprig/wprig/wiki/How-to-squash-commits) _commit_ 
+kamu dengan pesan _commit_ yang baik.
```

Comment:
Should we link `pesan _commit_ yang baik` with https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines?

Reply:
OK!